src/test/java/org/opensearch/flowframework/workflow/DeleteConnectorStepTests.java

```diff
@@ -58,7 +59,7 @@ public void testDeleteConnector() throws IOException, ExecutionException, Interr
             DeleteResponse output = new DeleteResponse(shardId, connectorIdArg, 1, 1, 1, true);
             actionListener.onResponse(output);
             return null;
-        }).when(machineLearningNodeClient).deleteConnector(any(String.class), any());
+        }).when(machineLearningNodeClient).deleteConnector(anyString(), anyActionListener());
```

Comment:
Nit:
```suggestion
        }).when(machineLearningNodeClient).deleteConnector(anyString(), any(ActionListener.class));
```

Reply:
> `any(ActionListener.class));`

This results in a compiler warning for unchecked cast. The only way to suppress that is to suppress it for the whole test.  I don't like compiler warnings.  So I made a nifty workaround that suppresses just the cast.